src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -0,0 +1,175 @@
+
+
+class TC_RVCCLEANM_2_1(MatterBaseTest):
+
+    async def read_mod_attribute_expect_success(self, endpoint, attribute):
+        cluster = Clusters.Objects.RvcCleanMode
+        return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
+
+    async def send_change_to_mode_cmd(self, newMode) -> Clusters.Objects.RvcCleanMode.Commands.ChangeToModeResponse:
+        ret = await self.send_single_cmd(cmd=Clusters.Objects.RvcCleanMode.Commands.ChangeToMode(newMode=newMode), endpoint=self.endpoint)
+        asserts.assert_true(type_matches(ret, Clusters.Objects.RvcCleanMode.Commands.ChangeToModeResponse),
+                            "Unexpected return type for ChangeToMode")
+        return ret
+
+    @async_test_body
+    async def test_TC_RVCCLEANM_2_1(self):
+
+        asserts.assert_true('PIXIT_ENDPOINT' in self.matter_test_config.global_test_params,
+                            "PIXIT_ENDPOINT must be included on the command line in "
+                            "the --int-arg flag as PIXIT_ENDPOINT:<endpoint>")
+        asserts.assert_true('PIXIT_MODEOK' in self.matter_test_config.global_test_params,
+                            "PIXIT_MODEOK must be included on the command line in "
+                            "the --int-arg flag as PIXIT_MODEOK:<mode id>")
+        asserts.assert_true('PIXIT_MODEFAIL' in self.matter_test_config.global_test_params,
+                            "PIXIT_MODEFAIL must be included on the command line in "
+                            "the --int-arg flag as PIXIT_MODEFAIL:<mode id>")
+
+        self.endpoint = self.matter_test_config.global_test_params['PIXIT_ENDPOINT']
+        self.modeok = self.matter_test_config.global_test_params['PIXIT_MODEOK']
+        self.modefail = self.matter_test_config.global_test_params['PIXIT_MODEFAIL']
+
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.A0000"), "RVCCLEANM.S.A0000 must be supported")
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.A0001"), "RVCCLEANM.S.A0001 must be supported")
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.C00.Rsp"), "RVCCLEANM.S.C00.Rsp must be supported")
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.C01.Tx"), "RVCCLEANM.S.C01.Tx must be supported")
+
+        attributes = Clusters.RvcCleanMode.Attributes
+
+        self.print_step(1, "Commissioning, already done")
+
+        modes = []
+
+        self.print_step(2, "Read SupportedModes attribute")
+        supported_modes = await self.read_mod_attribute_expect_success(endpoint=self.endpoint, attribute=attributes.SupportedModes)
+
+        logging.info("SupportedModes: %s" % (supported_modes))
+
+        asserts.assert_greater_equal(len(supported_modes), 2, "SupportedModes must have at least two entries!")
+
+        for m in supported_modes:
+            modes.append(m.mode)
+
+        self.print_step(3, "Read CurrentMode attribute")
```

Comment:
There is a typo for this step number in the [test plan](https://github.com/CHIP-Specifications/chip-test-plans/blob/14e25b555da72ead79b65376b35921b9d7b19bbf/src/cluster/modebase_common.adoc#L368). Is there an issue against this to get fixed?